docs/security/simple-ldap-variables.md

```diff
@@ -1,100 +1,158 @@
-# Simple LDAP Variables
-
-The following variables are static and can only be changed at runtime.
```

Comment:
Sorry, my comment here was incorrect previously: some variables are static, just not all of them - previously everything was grouped under the "static variables" section. Generally the connection variables (host, port, ssl, ...) are static, but everything else (including most new variables) are dynamic.

Reply:
Checked against MySQL system variables.